CONTRIBUTING.md

```diff
@@ -190,19 +190,62 @@ make
 
 ### Compiler Options
 
-By default, when building project with no options, next static libraries are generated
+By default, when building project with no options, next static libraries are generated:
```

Comment:
The original comment had a typo, it should say "no static..."

Reply:
I think the intent is to list the set of libraries that generated when no options are provided.

It isn't true that "no static libraries are generated". The libraries listed below (az_core, az_iot, etc.) do get generated.

Reply:
```suggestion
By default, when building the project with no options, the following static libraries are generated:
```